rkt/app_sandbox.go

```diff
@@ -33,6 +34,8 @@ var (
 		Long:  "Initializes an empty pod having no applications.",
 		Run:   runWrapper(runAppSandbox),
 	}
+	flagAnnotations kvMap
```

Comment:
I also added a separate package to do this, `pkg/flag/pairlist.go` - if you wanted to, you could use that.